Specs/createScene.js

```diff
@@ -58,6 +60,10 @@ define([
             return scene.context.readPixels();
         };
 
+        scene.pickForSpecs = function() {
```

Comment:
In a future PR, you can probably replace all `scene.pick` calls in the tests with this.